### DISCUSSION TITLE
Rate too high - Erfahrungswerte MaxRate / AbsolutChange

### DISCUSSION THREAD

**Discussion by pfeifferch** [jomjol/AI-on-the-edge-device#2525] [Q&A]
Hallo Community

Meine AI hat gemotzt, der Verbrauch wäre zu hoch. 
Waren knapp 60 Liter laut Log. 

NEben der großen Badewanne (Nutzinhalt: ca.195 l) gibt es noch eine Entkalkungsanlage, welche vorzugsweise in den frühen Morgenstunden ihr Spülprogrammfährt (bzw auch früher wenn die  Kubikmeter durch sind). 
Da knallt 2x 2Minuten auch ordentlich was durch die Leitung, Zahlen habe ich aber keine...

Hat jemand Erfahrungswerte 
MaxRate AbsolutChange?

**Comment by caco3** [comment]
Schau dir mal die Grafik an und gehe Faktor 1.5..2 über den maximalen Wert.

Da muss man sich etwas herantasten.
